libs/fn/src/lib/cdk/focusable-list/focusable-list.directive.ts

```diff
@@ -0,0 +1,43 @@
+import { AfterViewInit, ContentChildren, Directive, Input, QueryList } from '@angular/core';
+import { map, startWith, takeUntil, tap } from 'rxjs/operators';
+import { FocusableItem, FocusableListService } from './focusable-list.service';
+import { FocusableItemDirective } from './focusable-item.directive';
+import { DestroyedBehavior } from '../common-behaviors/destroyed-behavior';
+
+@Directive({
+    selector: '[fnFocusableList]',
+    exportAs: 'fnFocusableList',
+    providers: [FocusableListService, DestroyedBehavior]
+})
+export class FocusableListDirective implements AfterViewInit {
+    @Input()
+    navigationDirection: 'horizontal' | 'vertical' = 'vertical';
+
+    @Input()
+    contentDirection: 'ltr' | 'rtl' | null = 'ltr';
```

Comment:
Will we be using something similar to fundamental-ngx's RtlService for the FN library? BidiModule from Angular CDK? Would be nice for directionality to come automatically and not as an input